### DISCUSSION TITLE
sops file configuration problem

### DISCUSSION THREAD

**Discussion by aayushi-2001** [egovernments/Digit-Core#462] [Bugs]
FULL INSTALLTION OF DIGIT
I AM TRYING TO INSTALL IT THORUGH AWS ACCOUNT
I am using windows 11
https://digit-discuss.atlassian.net/wiki/spaces/DOPS/pages/2335997953/How+to+install+UPYOG+on+AWS

I am facing this problem in step 12 -"go run digit_installer.go" after choosing yes for actual deployement I am facing this problem
Please enter the cluster context to be used from the avaliable contexts: eks_demo
Switched to context "eks_demo".
v DIGIT
Use the arrow keys to navigate: ↓ ↑ → ←
v digit-v2.7
Use the arrow keys to navigate: ↓ ↑ → ←
v m_pgr
v Exit
2024/07/10 12:49:45 egov-demo.yaml
2024/07/10 12:49:45 nginx-ingress-controller.yaml
Use the arrow keys to navigate: ↓ ↑ → ←
v egov-demo
v No
v Yes

2024/07/10 12:50:49 C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code
2024/07/10 12:50:49 Helm Directory - C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code/helm
C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code\helm\charts\cluster-configs
2024/07/10 12:50:49 ------------------------------------ DEPLOYING CLUSTER CONFIGS ------------------------------------
Before TrimmingC:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code/helm
path to sops file: C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-cod
Sops file not found
2024/07/10 12:50:49 helm template -f C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code\environments\egov-demo.yaml --output-dir C:\Users\91934\AppData\Local\Temp\helm-3633823824 -f C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-code\environments\egov-demo-secrets.yaml .
CreateFile C:\Users\91934\Documents\digit\DIGIT-DevOps\config-as-cod/.sops.yaml: The system cannot find the path specified.
2024/07/10 12:50:50 exit status 1: Error: template: cluster-configs/templates/ingress/root-ingress.yaml:10:13: executing "cluster-configs/templates/ingress/root-ingress.yaml" at <$.Values.global.setup>: nil pointer evaluating interface {}.setup

Use --debug flag to render out invalid YAML

panic: exit status 1: Error: template: cluster-configs/templates/ingress/root-ingress.yaml:10:13: executing "cluster-configs/templates/ingress/root-ingress.yaml" at <$.Values.global.setup>: nil pointer evaluating interface {}.setup

Use --debug flag to render out invalid YAML

goroutine 1 [running]:
log.Panicln({0xc00007b5e0?, 0x10e?, 0xff?})
        C:/Program Files/Go/src/log/log.go:446 +0x5a
deployer/pkg/cmd/deployer.execCommandRaw({0xc000226ea0?, 0x0?}, {0xc00001aae0, 0x56}, 0x0)
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/pkg/cmd/deployer/deployer.go:259 +0x309
deployer/pkg/cmd/deployer.execCommand(...)
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/pkg/cmd/deployer/deployer.go:235
deployer/pkg/cmd/deployer.deployClusterConfigs(0xc000094300, {0xc0000a8340, 0x3f}, {0xc0002348a0, 0x56}, {0xc0002349c0, 0x5e})
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/pkg/cmd/deployer/deployer.go:165 +0x899
deployer/pkg/cmd/deployer.DeployCharts({{0x78f566, 0x14}, {0xc00007c700, 0x672}, {0xc00000a120, 0x9}, 0x1, 0x0})
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/pkg/cmd/deployer/deployer.go:27 +0x39b
deployer/cmd.init.func2(0xc000122300?, {0xc00009a0c0?, 0x4?, 0x789fe8?})
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/cmd/deploy.go:51 +0x85
github.com/spf13/cobra.(*Command).execute(0xa03140, {0xc00009a040, 0x4, 0x4})
        C:/Users/91934/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x684
github.com/spf13/cobra.(*Command).ExecuteC(0xa033c0)
        C:/Users/91934/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2df
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/91934/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
deployer/cmd.Execute()
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/cmd/root.go:48 +0x1a
main.main()
        C:/Users/91934/Documents/digit/DIGIT-DevOps/deploy-as-code/deployer/main.go:21 +0xf
exit status 2
2024/07/10 12:50:50 cmd.Run() failed with exit status 1
exit status 1

I am facing an error like this for sops configuration which you all faced but that park is skipped in the video. Please help me out

**Comment by naresh-egov** [comment]
@aayushi-2001 Are you trying to install UPYOG or DIGIT-Core?
